cvpy/tests/test_imagetable.py

```diff
@@ -266,6 +266,139 @@ def test_imagetable_from_table_custom_col_names(self):
 
         assert image_table.table.name == cas_table.name
         assert type(image_table) == NaturalImageTable
+    
+    # Load client images and server images using load_client_images and loadImages and compare results
+    def test_imagetable_load_client_images(self):
+        # For determining if we are working from a local machine or the Jenkins server
+        # and comparing images for verification
+        from PIL import ImageChops, ImageStat
+        from os.path import exists
+
+        # Path to the directory the function will load images from
+        path = "images/"
+
+        # If path exists, we are on the Jenkins server
+        if exists(path):
+            # Images are determined to be similar if the average pixel value of all channels
+            # are less than three percent of their maximum value.
+            validThreshold = 255 * 0.03
+
+            # Test loading images fron both client and server. One will have a valid path and
+            # return value to test the type, the other will try to load from a nonexistent directory
+            # and the last will load images from the server
+            natural = ImageTable.load_client_images(
+                output_table_parms={'name': 'imgsClientDirectory', 'caslib': 'CASUSER(user)'},
+                path=path,
+                connection=self.s,
+                subdirs=False
+            )
+            ImageTable.load_client_images(
+                output_table_parms={'name': 'imgsClientEmpty', 'caslib': 'CASUSER(user)'},
+                path='path/does/not/exist',
+                connection=self.s,
+                subdirs=False
+            )
```

Comment:
It seems like you have three test cases under test_imagetable_load_client_images function. We should have one test case per function. 

Separate the test cases into different functions. e.g. 
 
test_imagetable_load_client_images
test_imagetable_load_client_images_invalid_path
...